tests/test_ideogram.py

```diff
@@ -227,16 +226,16 @@ def assert_callback(prop_value, nclicks, input_value):
     driver = dash_threaded.driver
 
     # assert 22 chromosomes + X and Y chromosomes
-    num_chromosoms = len(wait_for_elements_by_css_selector(driver, '.chromosome'))
-    assert num_chromosoms == 24
+    chromosomes = wait_for_elements_by_css_selector(driver, '.chromosome')
```

Comment:
Is there a particular reason that you changed these? (I'm just wondering if it somehow speeds things up, or ensures that we have a result before trying to test the assertion)

Reply:
I think it is to standardize the tests?

Reply:
For both reasons: standardize and get elements before taking their length (for possible debugging too).